akka-typed/src/main/scala/akka/typed/javadsl/Actor.scala

```diff
@@ -202,6 +201,15 @@ object Actor {
   }
 
   /**
+   * Behavior that waits for all child actors to stop and then
+   * runs some cleanup before stopping.
+   */
```

Comment:
Clarify that it actually stops them and awaits for them, not just awaits for it

Reply:
done